### PR TITLE
SDK-1390: Update travis and sonar-project.properties

### PR DESCRIPTION
**Pull request by MrBurtyyy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,11 @@ dist: xenial
   
 # Only clone the most recent commit.
 git:
-  depth: 1
+  depth: false
```
Comment (echarrod) [inline .travis.yml]: Do we need this so it can analyse the code coverage and issues of everything in the PR, rather than just the last commit?

Comment (MrBurtyyy) [reply .travis.yml]: Yep exactly

File: .travis.yml [hunk 2]
```diff
@@ -49,7 +51,18 @@ jobs:
...
+      python: "3.6.1"
+      addons:
+        sonarcloud:
+          organization: "getyoti"
+      install:
+        - pip install -r requirements.txt
+        - pip install -e .[dev]
+      script:
+        - pytest --cov=yoti_python_sdk yoti_python_sdk/tests --cov-report=xml:coverage-reports/coverage-new.xml
+        - sed -i 's+<source>.*</source>+<source>/home/travis/build/getyoti/yoti-python-sdk/yoti_python_sdk</source>+g' coverage-reports/coverage-new.xml
```
Comment (echarrod) [inline .travis.yml]: How come we need this command?

File: requirements.in [hunk 3]
```diff
@@ -7,6 +7,7 @@ itsdangerous==0.24
 pbr==1.10.0
 protobuf==3.7.0
 pyopenssl==18.0.0
+PyYAML==5.2 # Support for Python 3.4
```
Comment (echarrod) [inline requirements.in]: Support for Python 3.4 working with what - coverage reporting?

File: sonar-project.properties [hunk 4]
```diff
@@ -1,7 +1,9 @@
-sonar.projectKey = yoti-web-sdk:python
-sonar.projectName = python-sdk
+sonar.host.url = https://sonarcloud.io
+sonar.organization = getyoti
+sonar.projectKey = getyoti:python
+sonar.projectName = Python SDK
 sonar.projectVersion = 2.10.0
-sonar.exclusions=yoti_python_sdk/tests/**,examples/**
+sonar.exclusions = yoti_python_sdk/tests/**,examples/**
```
Comment (echarrod) [inline sonar-project.properties]: Should we also be excluding the (protobuf/) `attribute_public_api`, `common_public_api` and `share_public_api` folders?

**Comment by echarrod** [comment]
Presume it's just showing `No Coverage information` as there's no Python files to cover in this PR?

File: .travis.yml (hunk 2 above)
Comment (MrBurtyyy) [reply .travis.yml]: Pytest generates relative paths for the files, but sonar-scanner needs full paths to link the indexed files to the code coverage

File: requirements.in (hunk 3 above)
Comment (MrBurtyyy) [reply requirements.in]: No this is something that was raised as I was testing.  When I did a fresh install of the SDK, PyYAML was using a new version (5.3 I think) which specified it was not compatible with Python 3.4.  I've pinned it here to 5.2 which is compatible with Python 3.4.

File: .travis.yml (hunk 2 above)
Comment (echarrod) [reply .travis.yml]: Ok, bit messy but it this is the only way then this is fine

File: requirements.in (hunk 3 above)
Comment (echarrod) [reply requirements.in]: Ok, could we be slightly more descriptive then - something like `5.3+ does not support Python 3.4`?

**Comment by MrBurtyyy** [comment]
> Presume it's just showing `No Coverage information` as there's no Python files to cover in this PR?

Yes @echarrod

File: requirements.in (hunk 3 above)
Comment (MrBurtyyy) [reply requirements.in]: Yep 👍 I'll change that now

Comment (MrBurtyyy) [reply requirements.in]: Fixed in 113f774

**Pull request merged**